tests/anagram-all-fail/expected_results.json

```diff
@@ -0,0 +1,48 @@
+{
+  "tests": [
+    {
+      "test_code": "    h.assert_array_eq[String](failme, Anagram(\"bobs\", anagrams))",
+      "message": " Assert EQ failed.  Expected ([len=1: failme]) == ([len=0: ])",
+      "status": "fail",
+      "version": 2,
```

Comment:
This property is solely a top-level property.
```suggestion
```

Reply:
Will remove in the next iteration.